shared/gh/js/utils/gh.utils.time.js

```diff
@@ -212,7 +212,7 @@ define(['exports', 'gh.constants', 'moment', 'moment-timezone'], function(export
 
             // Convert the dates from ISO to UNIX for easier calculation
             var startDate = convertISODatetoUnixDate(moment.tz(term.start, 'Europe/London').add({'days': 2}).toISOString());
-            var endDate = convertISODatetoUnixDate(moment.tz(term.end, 'Europe/London').subtract({'days': 2}).toISOString());
+            var endDate = convertISODatetoUnixDate(moment.tz(term.end, 'Europe/London').subtract({'days': 2}).hours(23).minutes(59).seconds(59).toISOString());
```

Comment:
Wouldn't it be easier to just add a day to this instead ? In practice, the result would probably be the same.